README.md

```diff
@@ -13,12 +13,24 @@ Easily setup a service level objective using prometheus, based on lessons from t
 
 # Generating
 
-Lookup [slo_example.yml](./slo_example.yml) to parametize SLO and generate prometheus rules running
+Lookup [slo_example.yml](./examples/slo_example.yml) to parametize SLO and generate prometheus rules running
```

Comment:
'Look the file [slo_example.yml](./examples/slo_example.yml) to see how to parametrize SLOs and generate Prometheus rules by running the following command:'